### DISCUSSION TITLE
Database set wrong account.

### DISCUSSION THREAD

**Discussion by GoodDay360** [bs-community/blessing-skin-server#402] [Q&A] [answered]
When first setup I put different hostname and username.... But when it finished setup it show this error.  
![AFAFE7FB-C94C-49F1-9C3C-66868AEE64CD](https://user-images.githubusercontent.com/59399625/165689602-254fca0d-ab9b-416a-acd9-ee9b25b9e9ef.png)

**Comment by mochaaP** [comment, marked as answer]
Please modify `.env` manually.
